WooCommerce/Classes/ViewRelated/Orders/Order Creation/Synchronizer/RemoteOrderSynchronizer.swift

```diff
@@ -72,4 +73,26 @@ private extension RemoteOrderSynchronizer {
             self?.baseSyncStatus = baseStatus
         }
     }
+
+    /// Updates the underlying order as inputs are received.
+    ///
+    func bindInputs() {
+        setStatus.withLatestFrom(orderPublisher)
+            .map { newStatus, order in
+                order.copy(status: newStatus)
+            }
+            .assign(to: &$order)
+
+        setProduct.withLatestFrom(orderPublisher)
+            .map { productInput, order in
+                ProductInputTransformer.update(input: productInput, on: order)
+            }
+            .assign(to: &$order)
+
+        setAddresses.withLatestFrom(orderPublisher)
+            .map { addressesInput, order in
+                order.copy(billingAddress: addressesInput?.billing, shippingAddress: addressesInput?.shipping)
+            }
+            .assign(to: &$order)
```

Comment:
Should we add `setShipping` binding that already exists in `LocalOrderSynchronizer`?

Reply:
I didn't add it yet because I haven't confirmed that the binding properly supports removing a fee once the order has been created. But I just realized I that it is the same for dealing an item(It does not work yet) so I'll move it over too!

Reply:
Added https://github.com/woocommerce/woocommerce-ios/pull/6208/commits/c96c854b148d9961326631d6ea90dc3d93efe7d6